src/main/java/ru/evotor/framework/receipt/ReceiptApi.kt

```diff
@@ -72,6 +72,7 @@ object ReceiptApi {
 
 
     @JvmStatic
+    @Deprecated("Use getProductsByBarcode from InventoryApi")
```

Comment:
```suggestion
    @Deprecated(
        message = "Use getProductsByBarcode from InventoryApi",
        replaceWith = ReplaceWith(expression = "InventoryApi.getProductsByBarcode(context, barcode)", imports = ["ru.evotor.framework.inventory.InventoryApi"])
    )
```
Можно добавить `replaceWith`, чтобы в контекстном меню была возможность одним кликом заменить старый метод на новый

![image](https://github.com/user-attachments/assets/8fa7a721-3dd0-48e0-b9ed-ab30fdfc7557)